dim_checks/moz-fx-data-shared-prod/telemetry/unified_metrics/dim_checks.yaml

```diff
@@ -0,0 +1,95 @@
+dim_config:
+  owner:
+    email: lvargas@mozilla.com
+    slack: lvargas
+  alerts_enabled:
+    enabled: true
+    notify:
+      channels:
+        - dim_test_run
+  tier: tier_1
+  dim_tests:
+    - type: table_row_count
+      params:
+        condition: ">= 1000"
```

Comment:
Is this per day or total?
Would it better be  ">= min(COUNT DISTINCT user_id) per day"?

Reply:
this is per day, just checking that we have at least 1000 rows in that partition. This is primary meant to be a check that we have some data in that partition.

Reply:
I recommend setting this value closer to the average/min daily value in order to alert for big drops or increases in data volume which otherwise may go unnoticed and require incident investigations.